### PR TITLE
Bioscanners 2 electric boogaloo

### PR DESCRIPTION
**Pull request by TsunamiAnt**
## About The Pull Request

The old PR was fucked, here's the actual new one all in one package.
See:
#10751
#10749
#10584

## Why It's Good For The Game

Same thing as all the other ones. Go read those. Or don't.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

image

image

image

</details>

## Changelog
:cl:
balance: Made health analyzers a bit weaker to compensate.
add: Added the Bioscanner, a full-body medical scanning machine that can be upgraded and takes time to scan people.
balance: removed advanced analyzer from the techtree and made them admin-only. Moved the normal one to tier 2.
del: removed medical scanning functionality from PDA cartridges. Reagent scanning stays.
del: removes health analyzers from the autolathe. Why was specialist medical tooling printable on public lathes again? You wouldn't download a jaws of life!
/:cl:

### CODE REVIEW

**Comment by TsunamiAnt** [comment]
I swear to god don't merge this till It's ready, I gotta test and make the maps.

**Comment by TsunamiAnt** [comment]
That went faster than expected. Please remember to testmerge before you merge it fully.

**Comment by Miliviu** [comment]
If it's ready to be merged add Testing Evidence, if it's still a wip make it a draft pr!

**Comment by TsunamiAnt** [comment]
> If it's ready to be merged add Testing Evidence, if it's still a wip make it a draft pr!

fine, but the evidence bites back!!

**Review by Rukofamicom — APPROVED** [review]
I still approve of this

👍

**Review by PestoVerde322 — APPROVED** [review]
Fine by me mapwise, approved.

**Comment by Rukofamicom** [comment]
Medbay seems to operate largely the same as before, with exception to abnormal injuries going to the scanner.

File: code/modules/research/designs/medical_designs.dm
Comment (PestoVerde322) [inline code/modules/research/designs/medical_designs.dm:174]: Honestly, i think Advanced Health analizers should be a diskette research rather than being gated off to admin only.

File: code/modules/research/designs/autolathe_designs.dm
Comment (PestoVerde322) [inline code/modules/research/designs/autolathe_designs.dm:546]: Stealh change, the health analizer is no longer obtainable via autolathe, opt to replace it with the health sensor (like for medbots)

File: code/game/machinery/bioscanner.dm
Comment (PestoVerde322) [inline code/game/machinery/bioscanner.dm:20]: What's exactly the time to scan on a standard bioscanner?

**Review by PestoVerde322 — CHANGES_REQUESTED** [review]
Req. changes for the Stealth change, address that.

File: code/modules/research/designs/autolathe_designs.dm
Comment (PestoVerde322) [reply code/modules/research/designs/autolathe_designs.dm:546]: Nevermind, the health sensor is there, just add the CL for the health analizer

**Comment by kiekatz** [comment]
regarding health analyzer changes... i have raised this point before. and ive personally not seen other unrelated pr's that fix it by lucky chance so i will add it here

health analyzing without a health analyzer in emergency situations outside of medical... is scuffed... **(OUTSIDE OF SIMPLIFIED BODY TARGETING)** you cant see what bodypart is injured. just that... they are injured by some damage type... the defacto MOST basic medical check that even the most untrained assistant should be able to preform in emergency situations...

since this pr has decided to tap into health analyzing. its only fair for it to add. the ability to see what bodypart is injured by what damage type when examining a person... not even exact numbers just vauge (lightly/moderately/badly) (bashed/burned/[insert damage type])

**Comment by Rukofamicom** [comment]
I maintain that health analyzers should be removed in favor of a good old fashioned examine text overhaul without exact numbers, but I know better than to push that right now.

**Comment by Miliviu** [comment]
> I maintain that health analyzers should be removed in favor of a good old fashioned examine text overhaul without exact numbers, but I know better than to push that right now.

If the text is limb localized I'd go ahead!

**Review by PestoVerde322 — APPROVED** [review]

**Comment by PestoVerde322** [comment]
CL change has been adressed.

**Comment by TsunamiAnt** [comment]
Yeah this is hella dead, I took some time to think about it and with the requested changes this is just useless.

**Pull request closed without merging**
